lib/api.js

```diff
@@ -0,0 +1,68 @@
+'use strict'; // eslint-disable-line strict
+
+const Config = require('./Config.js');
+const Logger = require('./Logger.js');
+
+class API {
+
+    /**
+     * This is the constructor of the Logger class. It takes optional
+     * configuration parameters, that allow to modify its behavior.
+     *
+     * @param {object} config         - A configuration object for werelogs.
+     * @param {string} config.level   - The string name of the logging level
```

Comment:
We already have typing information in `{string}`, no need to repeat that in the doc comment.